packages.go

```diff
@@ -164,7 +164,7 @@ func (pkgDefs *PackagesDefinitions) parseTypesFromFile(astFile *ast.File, packag
 
 func (pkgDefs *PackagesDefinitions) parseFunctionScopedTypesFromFile(astFile *ast.File, packagePath string, parsedSchemas map[*TypeSpecDef]*Schema) {
 	for _, astDeclaration := range astFile.Decls {
-		if funcDeclaration, ok := astDeclaration.(*ast.FuncDecl); ok {
+		if funcDeclaration, ok := astDeclaration.(*ast.FuncDecl); ok && funcDeclaration.Body != nil {
 			for _, stmt := range funcDeclaration.Body.List {
```

Comment:
It's more readable with 

```go
funcDeclaration, ok := astDeclaration.(*ast.FuncDecl);
if  ok && funcDeclaration.Body != nil {
```

Reply:
Hmm, I checked other functions some of areas used this way some not. I can change it.

Reply:
Changed ✔️